Author: Bruno Pagani
URL: https://github.com/archlinux/svntogit-community/commit/0368876157edf0b25cb0ae1ee1eb99d155bf101f
Summary: Drop strict Qt version check. Qt is ABI stable and jami is not using any private API
----
--- a/src/app/mainapplication.cpp
+++ b/src/app/mainapplication.cpp
@@ -100,12 +100,6 @@ ScreenInfo::setCurrentFocusWindow(QWindow* window)
 MainApplication::MainApplication(int& argc, char** argv)
     : QApplication(argc, argv)
 {
-    const char* qtVersion = qVersion();
-    qInfo() << "Using Qt runtime version:" << qtVersion;
-    if (strncmp(qtVersion, QT_VERSION_STR, strnlen(qtVersion, sizeof qtVersion)) != 0) {
-        qFatal("Qt build version mismatch! %s", QT_VERSION_STR);
-    }
-
     parseArguments();
     QObject::connect(this, &QApplication::aboutToQuit, [this] { cleanup(); });
 }
